### DISCUSSION TITLE
Network Setup

### DISCUSSION THREAD

**Discussion by GitCat3** [fnuecke/oc2#142] [Q&A]
I am kinda confused on how to use setup-network.lua. when I run it and answer yes to setting up DHCP client, it says discovering three or so times and then fails. Can anyone give me some info on how to set up a network using this?

**Comment by fnuecke** [comment]
You need one DHCP server in your network. So for one machine you have to say "no" to DHCP *client*, setup a fixed IP, then answer "yes" when it asks you whether this machine should act as a DHCP *server*.

**Reply by GitCat3** [reply]
And then is there any setup needed for the other computers that will be a client?

**Reply by fnuecke** [reply]
For those you can then just say yes to DHCP client, and they should find the DHCP server automatically.

**Reply by GitCat3** [reply]
Alright thanks
